storage/internal/benchmarks/client_pool.go

```diff
@@ -175,8 +169,8 @@ func initializeHTTPClient(ctx context.Context, writeBufferSize, readBufferSize i
 		IdleConnTimeout:       90 * time.Second,
 		TLSHandshakeTimeout:   10 * time.Second,
 		ExpectContinueTimeout: 1 * time.Second,
-		WriteBufferSize:       writeBufferSize,
-		ReadBufferSize:        readBufferSize,
+		WriteBufferSize:       p.writeBufferSize,
+		ReadBufferSize:        p.readBufferSize,
 	}
```

Comment:
Unrelated to this PR but I realized that the setting of `MaxIdleConnsPerHost` that we do in google-api-go-client is missing here. We should discuss offline.

Reply:
We should set the default to `100` for sure rather than the net/http default, even if it's not a param for now. But it would be a relevant param for a high throughput workload as well.

Reply:
`100` is actually the http default and is what it's set to here.

Reply:
I think you are confused with `MaxIdleConns` maybe?

See https://pkg.go.dev/net/http#DefaultMaxIdleConnsPerHost and https://pkg.go.dev/net/http#Transport.MaxIdleConnsPerHost . It must be set explicitly or else the value will be 2, which is not good for perf under high load

Reply:
Yep, I got confused. Thanks for bringing this up! I set `MaxIdleConnsPerHost` to `100`